packages/react-data-grid/src/index.ts

```diff
@@ -1,16 +1,13 @@
 export { default, ReactDataGridProps } from './ReactDataGrid';
-export { default as rowComparer } from './common/utils/RowComparer';
 export { default as Cell } from './Cell';
 export { default as Row } from './Row';
 export { default as HeaderCell } from './HeaderCell';
-import * as _utils from './common/utils';
 import * as _helpers from './helpers';
 export * from './Columns';
 export * from './formatters';
 export * from './common/editors';
 export * from './common/enums';
 export * from './common/types';
 export {
-  _utils,
   _helpers
```

Comment:
Need to delete this as well

Reply:
Maybe we can start moving stuff off of the addons package.